backend/controllers/appointments/bulkAppointmentsUpload.js

```diff
@@ -63,11 +74,14 @@ const bulkAppointmentsUploadFactory = (csvParserService, offenderLoader, logErro
             [[], []]
           )
 
+          prisonersDetails = await getLocations(res.locals, prisonersDetails)
+
           const prisonerList = prisonersDetails.map(prisoner => ({
             bookingId: prisoner.bookingId,
             offenderNo: prisoner.offenderNo,
             firstName: prisoner.firstName,
             lastName: prisoner.lastName,
+            cellNo: prisoner.cellNo,
```

Comment:
You could do the location lookup as part of this, saving mapping over the list twice and modifying it:
```const prisonerList = prisonersDetails.map(prisoner => (
   const location = await elite2Api.getLocation(context, prisoner.assignedLivingUnitId)
   return {
        bookingId: prisoner.bookingId,
        bookingId: prisoner.bookingId,
        offenderNo: prisoner.offenderNo,
        offenderNo: prisoner.offenderNo,
        firstName: prisoner.firstName,
        firstName: prisoner.firstName,
        lastName: prisoner.lastName,
        lastName: prisoner.lastName,
        cellNo: location.description,
        }
        )
```

This will also eliminate the need to modify the loader, or have a separate call to get the locations.